rsatoolbox/data/computations.py

```diff
@@ -20,7 +21,7 @@ def average_dataset(dataset):
     return np.mean(dataset.measurements, axis=0)
 
 
-def average_dataset_by(dataset, by):
+def average_dataset_by_slow(dataset, by):
```

Comment:
I think if the slow one is not referenced anywhere you can just remove it. If we need to see how it used to be implemented we can use git history right?

Reply:
yes, of course, this will be gone when I merge this in